### DISCUSSION TITLE
Regarding non-academic entrants

### DISCUSSION THREAD

**Discussion by codepilot** [Xilinx/fpga24_routing_contest#11] [Q&A] [answered]
I am not currently associated with any academic institution, may I enter?

I use Vivado and RapidWright in my daily work.

**Comment by clavin-xlnx** [comment, marked as answer]
> I am not currently associated with any academic institution, may I enter?

Sure!  Being a student/academic is not a requirement.

> I use Vivado and RapidWright in my daily work.

That's great!  We'd love to hear more about what you use it for!

**Reply by codepilot** [reply]
Thanks, I sent an email to enter.

I use https://www.bittware.com/fpga/xup-p3r/ boards for cloud services, see https://reports.tolly.com/publications/221141.

I have patents in the field, https://patents.google.com/?inventor=daniel+kluss

I am using RapidWright specifically because routing times for vu9p are really long. I greatly appreciate the efforts to bring them down with this contest.
